projects/plugins/search/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"private": true,
+	"description": "A cloud-powered replacement for WordPress' search.",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "echo 'Not implemented.",
+		"build-js": "echo 'Not implemented.",
+		"build-production": "echo 'Not implemented.",
+		"build-production-js": "echo 'Not implemented.",
+		"clean": "true",
+		"distclean": "rm -rf node_modules && pnpm run clean",
+		"install-if-deps-outdated": "pnpm install --no-prod --frozen-lockfile",
+		"validate-es5": "eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore"
```

Comment:
We should probably remove `validate-es5` from the skeleton, we probably don't transpile down to ES5 anymore. See also p9dueE-3fG-p2.

Reply:
@anomiex Does any project still need it? I'll remove it from the skeleton and drop it from everything if we don't need it anywhere.

Reply:
Not that I know of. It'd be nice to have a modernized check based on the browsers list, but I haven't figured out a good way to make that happen yet. There's further in p9dueE-3fG-p2; maybe we should copy that into a public issue.

Reply:
Yeah that would be a good idea to move that into the public space. I'll move it over to https://github.com/Automattic/jetpack/pull/20817